app/components/card_component.rb

```diff
@@ -1,5 +1,16 @@
 class CardComponent < ApplicationComponent
-  HEADER_VARIANTS = {default: "p-2 sm:p-4", no_padding: ""}
+  attr_accessor :media
```

Comment:
I feel  like `header_media` is more specific, but not necessarily worth changing.

Reply:
I thought about using that name, but also didn't feel strongly enough about it to change it.🤷